### PR TITLE
Create style-guides-and-pattern-libraries-a-foundation-to-consistency

### PR DESCRIPTION
**Pull request by whitingx**
Write up of a recent talk given on Style Guides

### CODE REVIEW

**Comment by bradfrost** [comment]
Awesome thanks @whitingx!

File: _resourcetalk/style-guides-and-pattern-libraries-a-foundation-to-consistency.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
```
Comment (whitingx) [inline _resourcetalk/style-guides-and-pattern-libraries-a-foundation-to-consistency.md:1]: Not sure if it's a formatting error somewhere but the link for this talk redirects back to the 'talks' page.

cc/ @bradfrost @maban

**Comment by michaelthorne** [comment]
@whitingx the issue was with the `:` in the value of the title of the talk. You need to wrap the entire value in double quotes `"` or use the HTML character entity `&#58;` in place of the colon.

https://github.com/assemble/assemble/issues/21
http://stackoverflow.com/questions/8783705/how-could-should-i-state-colon-punctuation-in-a-yaml-file

**Comment by whitingx** [comment]
Thanks @michaelthorne :+1: sent a PR with this change.

**Comment by michaelthorne** [comment]
@whitingx no worries, I'd already updated it. :+1:

Not sure why the PR is on the previous version (before I edited it) of the file. :8ball:

**Pull request merged**